src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +43,54 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={handleChange}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {notFoundMovie && query && showErrorMessage && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames('button', 'is-light', {
+                'is-loading': !finished,
+              })}
+              disabled={Boolean(!query.length)}
```

Comment:
the logical operator will convert the value to the boolean type anyway 
```suggestion
              disabled={!query.length}
```